soap/src/java/com/zimbra/soap/mail/message/SetRecoveryAccountRequest.java

```diff
@@ -52,18 +53,25 @@ public static Op fromString(String s) throws ServiceException {
     private Op op;
 
     /**
-     * @zm-api-field-tag recoveryEmailAddress
-     * @zm-api-field-description recovery email address
+     * @zm-api-field-tag recoveryAccount
```

Comment:
Change this to email element

Reply:
change this to &lt;recoveryaccount channel="email">&lt;/recoveryaccount>
This should be list

Reply:
As per discussion, this change is not needed.